docs/excel/custom-functions-debugging.md

```diff
@@ -0,0 +1,90 @@
+7. Copy the URL for this new workbook.
+8. In VS Code, select **F5** to begin debugging. A prompt will appear, which asks for the URL of your document.
+9. Paste in the URL for your workbook.
+10. At this point, debugging is active. You can set breakpoints within VS Code.
+11. To see your breakpoints applied, you'll go back to the Edge window with your open workbook and insert your add-in.
+    - Select **Insert | Office Add-ins**. This will open a pop-up window.
+    - Select **Manage My Add-ins | Upload Add-in** in the new pop-up window.
+    - Browse for the manifest file of your add-in and select Upload.
+
+### Mac and Chrome
+
+1. Open your custom functions root project folder in VS Code.
+2. You may be prompted to install recommended extensions if you have based this project off of the latest yo office generator project. You should select install all, which will install [Debugger for Edge](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-edge) and [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extensions for VS Code. If you are not prompted to install these, install these extensions to VS Code manually.
+3. Select **Terminal | Run Task** and type or select Dev Server.
+4. Switch to Debug View selecting **View | Debug** or by entering **Ctrl+Shift+D**.
+5. From the Debug options, choose Office Online (Chrome).
+6. Open Excel Online using Chrome and create a new workbook.
+7. Copy the URL for this new workbook.
+8. In VS Code, select **F5** to begin debugging. A prompt will appear, which asks for the URL of your document.
+9. Paste in the URL for your workbook.
+10. At this point, debugging is active. You can set breakpoints within VS Code.
+11. To see your breakpoints applied, you'll go back to the Chrome window with your open workbook and insert your add-in.
+    - Select **Insert | Office Add-ins**. This will open a pop-up window.
+    - Select **Manage My Add-ins | Upload Add-in** in the new pop-up window.
+    - Browse for the manifest file of your add-in and select Upload.
+
+## Office Desktop
+
+### For Windows Or Mac
+
+1. Open your custom functions root project folder in VS Code.
+3. Select **Terminal | Run Task** and type or select Dev Server.
+4. Switch to Debug View selecting **View | Debug** or by entering **Ctrl+Shift+D**.
+5. From the Debug options, choose Office Desktop.
+6. Select **F5** to begin debugging. A new Excel workbook will pop up, with your add-in already sideloaded and ready to use.
+7. To debug, set breakpoints within VS Code.
+
+## Using the command line
+
+To debug using the command line, follow the same step sequences for platform and product, but replace VS Code terminal and debugging commands with statements below, depending on your needs:
+
+### Watch and build your project
+
+- `npm run watch`: builds for development and automatically rebuilds when a source file is saved.
```

Comment:
```suggestion
- `npm run watch`: builds for development and automatically rebuilds when a source file is saved
```

Reply:
Per MSTP, for word phrases in lists, we don't use periods at the end.

Reply:
Oh! Ok I had that one backwards in my head for some reason. Thanks!